db/migrate/20220226221224_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.integer :authorId
+      t.string :title
+      t.text :text
+      t.integer :comments_count
+      t.integer :likeCount
```

Comment:
[Optional] According to Ruby's best practice, Kindly use `snake_case` for methods and variables.